firefox_puppeteer/firefox_puppeteer/api/software_update.py

```diff
@@ -7,9 +7,9 @@
 
 import mozinfo
 
+from firefox_puppeteer.base import BaseLib
 from firefox_puppeteer.api.appinfo import AppInfo
 from firefox_puppeteer.api.prefs import Preferences
-from firefox_puppeteer.base import BaseLib
 
```

Comment:
So, I don't see the rhyme or reason of how these were ordered before or after. Shouldn't this be sorted alphabetically by package and then module?

Note that I won't block the review for this; it's not that big a deal.